fixtures/regressions/logging-callback-interface/src/lib.rs

```diff
@@ -6,7 +6,7 @@ use std::sync::{Once, RwLock};
 
 // Logger trait that the foreign code implements
 pub trait Logger: Sync + Send {
-    fn log_message(&self, message: String);
+    fn log_message(&self, message: String, level: String);
```

Comment:
String is more clear than `u8`, and seems like overkill introducing a UniFFI exported Enum.